static/sass/styles.scss

```diff
@@ -853,3 +853,12 @@ html {
 img.has-shadow {
   box-shadow: 0 0 1rem #e5e5e5;
 }
+
+// Style for status labels (similar to chips but non-clickable)
+.p-status-label--rounded {
+  background-color: #ddf2e0;
+  border-radius: 2rem;
+  padding-bottom: 0.051rem;
```

Comment:
please use the vanilla variables for this. We're about to change heading styles, if this is hardcoded here, it will look very broken very soon.

Reply:
please also subtract thevalue you're adding as padding-bottom from the margin-bottom to ensure the element remains aligned with the baseline grid.